Tests/Command/ConsumeCommandTest.php

```diff
@@ -16,10 +19,30 @@ class ConsumeCommandTest extends KernelTestCase
 
     protected $mockConsumer;
 
-    public function setMockConsumer($expirationCount)
+    public function setUp()
     {
         self::bootKernel();
+    }
+
+    public function setMockQueueDriver()
+    {
+        $msg = new QueueMessage(new Message());
+
+        $this->mockQueueDriver = $this
+            ->getMockBuilder(StompQueueDriver::class)
```

Comment:
In this case you can create the mock from `QueueDriverInterface`that is the interface shared

Reply:
Ok, I changed this to use the QueueDriverInterface.